### PR TITLE
column ordering fixes

### PR DESCRIPTION
**Pull request by steve-haar**

### CODE REVIEW

File: grid-demo/src/app/angular/column-ordering/custom-clr-column-ordering-column.directive.ts [hunk 1]
```diff
@@ -49,6 +49,19 @@ export class CustomClrColumnOrderingColumnDirective {
     }
   }
 
+  ngAfterViewInit(): void {
+    const separator = this.elementRef.nativeElement.querySelector('clr-dg-column-separator');
+    console.log(this.elementRef.nativeElement, 'sep');
```
Comment (kevinbuhmann) [inline grid-demo/src/app/angular/column-ordering/custom-clr-column-ordering-column.directive.ts:54]: Should this be removed?

File: grid-demo/src/app/angular/column-ordering/custom-clr-column-ordering-column.directive.ts [hunk 2]
```diff
@@ -49,6 +49,19 @@ export class CustomClrColumnOrderingColumnDirective {
...
+    if (separator) {
+      separator.addEventListener('mouseenter', () => {
+        this.cdkDrag.disabled = true;
+      });
+      separator.addEventListener('mouseleave', () => {
+        this.cdkDrag.disabled = false;
+      });
```
Comment (kevinbuhmann) [inline grid-demo/src/app/angular/column-ordering/custom-clr-column-ordering-column.directive.ts:61]: We should remove these listeners in `ngOnDestroy`. The easiest way I know would be use `fromEvent` and unsubscribe from the observable like normal.

Comment (kevinbuhmann) [inline grid-demo/src/app/angular/column-ordering/custom-clr-column-ordering-column.directive.ts:61]: We should remove these listeners in `ngOnDestroy`. The easiest way I know would be use `fromEvent` and unsubscribe from the observable like normal.

**Review by kevinbuhmann — CHANGES_REQUESTED** [review]
See comments.

File: grid-demo/src/app/angular/column-ordering/custom-clr-column-ordering-column.directive.ts (hunk 1 above)
Comment (steve-haar) [reply grid-demo/src/app/angular/column-ordering/custom-clr-column-ordering-column.directive.ts:54]: yes, removed

File: grid-demo/src/app/angular/column-ordering/custom-clr-column-ordering-column.directive.ts (hunk 2 above)
Comment (steve-haar) [reply grid-demo/src/app/angular/column-ordering/custom-clr-column-ordering-column.directive.ts]: yes, fixed

**Review by kevinbuhmann — APPROVED** [review]

**Pull request merged**